### PR TITLE
global: addition of changing url

### PR DESCRIPTION
**Pull request by drjova**
* Replaces location url when the ``pid`` is available.

Signed-off-by: Harris Tzovanakis <me@drjova.com>

### CODE REVIEW

File: src/invenio-records-js/controllers/InvenioRecordsCtrl.js [hunk 1]
```diff
@@ -369,6 +370,20 @@ function InvenioRecordsCtrl($scope, $rootScope, $q, $timeout,
...
+  /**
+    * Updating the location with the new ``pid``
+    * @memberof InvenioRecordsCtrl
+    * @function invenioRecordsLocationUpdated
+    * @param {Object} evt - The event object.
+    * @param {String} pid - The deposition id.
+    */
+  function invenioRecordsLocationUpdated(evt, pid) {
+    // FIXME: Make it configurable
+    $location.url('/deposit/' + pid);
```
Comment (drjova) [inline src/invenio-records-js/controllers/InvenioRecordsCtrl.js]: It would be better if we take the endpoint from the API's ``response``

**Comment by lnielsen** [comment]
will you rebase then I can merge

**Comment by drjova** [comment]
ping @lnielsen done :) ``html`` key in the ``links`` required if some wants to change the location

**Comment by drjova** [comment]
@lnielsen ready

**Pull request merged**